cluster-autoscaler-version.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+if [[ $# -eq 0 ]]
+then
+  echo 'USAGE: version.sh <kubernetes-version>'
+  exit 1
+fi
+
+VERSION=$(git \
+    -c 'versionsort.suffix=-' \
+    ls-remote \
+    --exit-code \
+    --tags \
+    --sort='v:refname' \
+    git@github.com:kubernetes/autoscaler.git \
```

Comment:
Would it be possible to use `https://github.com/kubernetes/autoscaler.git` instead?